base_img/etc/lighttpd/lighttpd.conf

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2017 by Delphix. All rights reserved.
+#
+server.document-root = "/var/www/"
+
+server.port = 80
```

Comment:
Most users will use HTTPS, so I'd suggest supporting 443 too. Also, keep in mind that the sysadmin can change the HTTP(S) ports, although that's not common.
More generally, you might want to reuse the security and web settings configured by sysadmins for the engine, such as the HTTPS certificate, HTTPS redirection, HSTS, ciphers, ports, etc.
I don't know much about lighttpd, but we already have nginx set up in the engine. If it's acceptable for the recovery environment to depend on nginx and the sysadmin-managed settings, you could make this recovery page work similarly to the two special pages we show during engine boot up and when the stack is down.